packages/core/FieldElement.h

```diff
@@ -102,7 +102,7 @@ FieldElement<T>::FieldElement(T default_value):
  *----------------------------------------------------------------------------*/
 template <class T>
 FieldElement<T>::FieldElement():
-    Field(ELEMENT, getImpliedEncoding<T>())
+    Field(ELEMENT, getImpliedEncoding<T>()), value()
```

Comment:
I prefer to avoid artificially initializing variables because that prevents tools like valgrind and address sanitizer from catching my mistakes.